src/components/register.component.js

```diff
@@ -127,7 +130,7 @@ const Register = () => {
                     borderBottom: `1px solid red`,
```

Comment:
First IF, where logic for bad pass is, must be `pass.length < 3 || pass !== repeatPass`

Reply:
done .